resteasy-genericresponse/src/test/scala/com/cerner/beadledom/resteasy/BuiltGenericResponseSpec.scala

```diff
@@ -436,5 +440,30 @@ class BuiltGenericResponseSpec
         verify(rawResponse).getHeaderString("header")
       }
     }
+
+    describe("#toString") {
+      it("returns String with all fields included") {
+        val rawResponse = mock[Response]
+        when(rawResponse.getStatus).thenReturn(200)
+        when(rawResponse.getMediaType).thenReturn(MediaType.APPLICATION_JSON_TYPE)
+        when(rawResponse.getDate).thenReturn(new Date(0L))
```

Comment:
Looks like the dates are failing on Travis due to timezone differences. Might be best to just get the expected dates programmatically as well.